examples/with-graphql-hasura/app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "nextjs-hasura-example",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "test": "NODE_OPTIONS='--inspect' jest --watch --runInBand --testPathIgnorePatterns=[]"
+  },
+  "dependencies": {
+    "@apollo/client": "^3.3.6",
+    "graphql": "^15.4.0",
+    "next": "10.0.4",
+    "react": "17.0.1",
+    "react-dom": "17.0.1"
+  },
+  "devDependencies": {
+    "@types/node": "^14.14.16",
+    "@types/react": "^17.0.0",
+    "typescript": "^4.1.3"
```

Comment:
Let's remove all the TypeScript stuff. It's not really being utilized, and it's more config/setup for an example. I err on the side of more simple examples, and then can add TypeScript if necessary.